### PR TITLE
Local env setup for linux

### PR DESCRIPTION
**Pull request by alsmk**
## Description

[#9281](https://github.com/opencrvs/opencrvs-core/issues/9281)
Added a local virtual machine setup for testing Ansible playbooks locally (on Linux )

## Checklist

- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [x] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: infrastructure/local-development/local.linux.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (rikukissa) [inline infrastructure/local-development/local.linux.yml:1]: Let's call this file a `local.linux.yml.template`. Please update the @todo commends accordingly. Not sure if we even need them if user never has to touch the file manually

Comment (rikukissa) [reply infrastructure/local-development/local.linux.yml:1]: Also make sure the provision script doesn't change the template but only copies it as a new file `local.linux.yml`

**Review by rikukissa — APPROVED** [review]

**Review by Zangetsu101 — APPROVED** [review]
